patchwork/templates/patchwork/series-list.html

```diff
@@ -0,0 +1,98 @@
+{% extends "base.html" %}
+
+{% load person %}
+{% load static %}
+
+{% block title %}{{project.name}}{% endblock %}
+{% block series_active %}active{% endblock %}
+
+{% block body %}
+
+{% load person %}
+{% load listurl %}
+{% load patch %}
+{% load project %}
+{% load static %}
+
+{% include "patchwork/partials/pagination.html" %}
+
+<input type="hidden" name="form" value="serieslistform"/>
+<input type="hidden" name="project" value="{{project.id}}"/>
+
+<table id="serieslist" class="table table-hover table-extra-condensed table-striped pw-list" data-toggle="checkboxes" data-range="true">
+  <thead>
+    <tr>
+{% if user.is_authenticated and user.profile.show_ids %}
+      <th>
+        ID
+      </th>
+{% endif %}
+
+      <th>
+        <span class="colinactive">Series</span>
+      </th>
+
+      <th>
+        Version
+      </th>
+
+      <th>
+        <span class="colinactive">Cover Letter</span>
+      </th>
+
+      <th>
+        <span class="colinactive">Patches</span>
+      </th>
+
+      <th>
+        <span class="colinactive">Date</span>
+      </th>
+
+      <th>
+        <span class="colinactive">Submitter</span>
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+{% for series in page %}
+    <tr id="series_row:{{series.id}}">
+{% if user.is_authenticated and user.profile.show_ids %}
+      <td>
+        <button type="button" class="btn btn-xs btn-copy" data-clipboard-text="{{ series.id }}" title="Copy to Clipboard">
+          {{ series.id }}
+        </button>
+      </td>
+{% endif %}
+      <td>
+        <a href="{% url 'series-detail' project_id=project.linkname series_id=series.id %}">
+          {{ series.name|default:"[no subject]"|truncatechars:100 }}
+        </a>
```

Comment:
This shouldn't be present in the first commit since the corresponding view doesn't exist yet.

![Screen Shot 2024-11-01 at 11 03 09](https://github.com/user-attachments/assets/0470bec2-3730-4333-b7a1-0e0359e5dfff)

Can you drop the `<a>` tags and just print the name here. You can re-add the `<a>` tags in the commit that adds the `series-detail` view.